utils/threshAlgos.py

```diff
@@ -1452,7 +1452,7 @@ def getDetName(entry):
                 # channelor case
                 if perchannel == False :
 	            dict_dacInflectPts[dacName][ohKey][vfat] = findInflectionPts(dict_Rate1DVsDACNameX[dacName][ohKey][vfat])
-                    inflectTable.append([ohKey, vfat, dict_dacInflectPts[dacName][ohKey][vfat][0][0] ])
+                    inflectTable.append([ohKey, vfat, dict_dacInflectPts[dacName][ohKey][vfat][0] ])
```

Comment:
I guess this fixes the same bug as the one described in https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/268?

Reply:
Yeah, now that it is restricted to have one index, the data structure changed a bit.